providers/dovetail.go

```diff
@@ -0,0 +1,41 @@
+package providers
+
+const Dovetail Provider = "dovetail"
+
+func init() {
+	SetInfo(Dovetail, ProviderInfo{
+		DisplayName: "Dovetail",
+		AuthType:    ApiKey,
+		BaseURL:     "https://dovetail.com",
```

Comment:
Any reason this isn't `https://dovetail.com/api`?

Reply:
No specific reason. I'll check with the docs and update it.